installer.sh

```
@@ -26,25 +26,25 @@
       export DEBIAN_FRONTEND=noninteractive
       cmd=$1
       shift
-      $SUDO apt-get $cmd -qq -o=Dpkg::Use-Pty=0 $@
+      $SUDO apt-get $cmd -qq -o=Dpkg::Use-Pty=0 $@ # FIXME
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)